src/model/Series.ts

```diff
@@ -599,9 +618,12 @@ class SeriesModel<Opt extends SeriesOption = SeriesOption> extends ComponentMode
             }
         }
         else if (selectedMode === 'single' || selectedMode === true) {
+            if (zrUtil.isString(option.selectedMap)) {
+                option.selectedMap = {};
+            }
```

Comment:
This seems unnecessary as the following lines 626-628 will always override this assignment. Right?

Reply:
resolved